GeometryService/src/STMMaker.cc

```diff
@@ -311,9 +379,11 @@ namespace mu2e {
     _magnetHoleHalfWidth       = _config.getDouble("stm.magnet.holeHalfWidth");
     _magnetHoleHalfHeight      = _config.getDouble("stm.magnet.holeHalfHeight");    
     _magnetMaterial            = _config.getString("stm.magnet.material");
-    _magnetField               = _config.getDouble("stm.magnet.field");
-    //_magnetFieldVisible        = _config.getBool(  "stm.magnet.fieldVisible",false);
-    _magnetFieldVisible        = geomOptions->isVisible("stmMagnetField");
+    _magnetField               = _config.getDouble("stm.magnet.field",0.0);
+    _magnetFieldx               = _config.getDouble("stm.magnet.fieldx",0.0);
+    _magnetFieldy               = _config.getDouble("stm.magnet.fieldy",0.0);
+    _magnetFieldz               = _config.getDouble("stm.magnet.fieldz",0.0);
+    _magnetFieldVisible        = _config.getBool(  "stm.magnet.fieldVisible",false);
```

Comment:
Should these all have default values? Could this lead to problems if one forgets to set something, where an error should be thrown?